CHANGELOG.unreleased.md

```diff
@@ -6,7 +6,10 @@
 
 ### Bug fixes
 
+- [SDN Controller] Do not choose physical PIF without IP configuration for tunnels.
```

Comment:
Missing PR link.

Reply:
Did this PR refer to an issue?

Reply:
Adding the PR link.
No issue refers to this, shall i create one?

Reply:
No need to create it.

Reply:
Is there a support ticket related to this?

If that's the case, you can reference it with this syntax: `xoa-support#42`.

Reply:
Done.